tests/testthat/test-calculate_age.R

```diff
@@ -0,0 +1,98 @@
+data <- readRDS(system.file("extdata", "test_df.RDS", package = "cleanepi"))
+
+test_that("calculate_age works", {
```

Comment:
Could you use more informative descriptions, for example what I'm suggesting? it'll make it easier to see what's different between the different blocks.
```suggestion
test_that("calculate_age works when age_remainder_unit is days", {
```